packages/providers/package.json

```diff
@@ -22,7 +22,8 @@
     "build": "npm-run-all -l clean -p build:esm"
   },
   "dependencies": {
-    "@myst-theme/common": "^1.0.0"
+    "@myst-theme/common": "^1.0.0",
+    "@myst-theme/search": "^1.0.0"
```

Comment:
I think just `devDependencies`, as we are only doing types?

https://github.com/jupyter-book/myst-theme/blob/main/packages/providers/src/search.tsx#L2

I think in future moving search/xref/config types to spec or common would be ideal, as that brings the contract, but none of the code.

Reply:
These types are exposed in the signatures exported by this package, so they're production.